server/test/testauth.js

```diff
@@ -0,0 +1,75 @@
+process.env.NODE_ENV = 'test';
```

Comment:
put this in the test script : `"process.env.NODE_ENV = 'test'  && then the script you have"`

Reply:
actually , there was an error while moving the script